support-frontend/assets/helpers/redux/checkout/personalDetails/thunks.ts

```diff
@@ -33,10 +33,12 @@ export const getUserTypeFromIdentity = createAsyncThunk<
 		condition: (email, thunkApi) => {
 			const { userTypeFromIdentityResponse, isSignedIn } =
 				thunkApi.getState().page.checkoutForm.personalDetails;
+			const { isStorybookUser } = thunkApi.getState().page.user;
+
```

Comment:
I don't feel great about effectively baking our Storybook dependency into real application code- I think we should simply avoid running this request in the Storybook context- but I appreciate that might require too significant a refactor at this point.

Reply:
This sounds like a great approach, will look into avoiding running this request in the Storybook context